src/main/resources/assets/chisel/lang/ko_KR.lang

```diff
@@ -1,538 +1,889 @@
+tile.technical.4.desc=환풍기 (빠름)
+tile.technical.5.desc=내부 파이프 (소형)
+tile.technical.6.desc=환풍기 (꺼짐)
+tile.technical.7.desc=통풍구
+tile.technical.8.desc=빛나는 통풍구
+tile.technical.9.desc=절연제
+tile.technical.10.desc=기어와 플라이휠
+tile.technical.11.desc=전선
+tile.technical.12.desc=녹슨 패널
+tile.technical.13.desc=창살
+tile.technical.14.desc=환풍기 (불량)
+tile.technical.15.desc=녹슨 창살
+tile.technical3.0.desc=나는 거대한 팬의 광팬이에요!
+tile.technical3.1.desc=산업 강화판
+
+# 금속 블록 (Metal Blocks)
+tile.chisel.aluminumblock.name=알루미늄 블록
+tile.chisel.bronzeblock.name=청동 블록
+tile.chisel.copperblock.name=구리 블록
+tile.chisel.gold2.name=금 블록
+tile.chisel.iron2.name=철 블록
+tile.chisel.leadblock.name=납 블록
+tile.chisel.silverblock.name=은 블록
+tile.chisel.steelblock.name=강철 블록
+tile.chisel.tinblock.name=주석 블록
+tile.chisel.uraniumblock.name=우라늄 블록
+tile.metalOre.0.desc=경고 줄무늬
+tile.metalOre.1.desc=선박 화물
+tile.metalOre.2.desc=써멀 익스펜션
+#tile.metalOre.3.desc=An Old Relic from the land of Oneteufyv
+tile.metalOre.4.desc=테두리를 두른 블록
+tile.metalOre.5.desc=화려한 볼트 판
+
+# 선물 상자 (Present Chest)
+tile.chisel.present.name=선물 상자
+tile.chisel.present.desc=예쁜 선물 상자
+
+# 벽돌 (Brick)
+tile.chisel.brickCustom.name=벽돌
+tile.brickCustom.1.desc=대형 벽돌
+tile.brickCustom.2.desc=모르타르가 없는 벽돌
+tile.brickCustom.3.desc=다양한 벽돌
+tile.brickCustom.5.desc=오래된 벽돌
+tile.brickCustom.6.desc=노란 벽돌
+
+#Chisel 1
+
+# 조약돌 (Cobblestone Blocks)
 tile.chisel.cobblestone.name=조약돌
-tile.cobblestone.0.desc=줄무늬 조약돌 벽돌
+tile.cobblestone.0.desc=조약돌 벽돌
 tile.cobblestone.1.desc=섬세한 조약돌 벽돌
 tile.cobblestone.2.desc=작은 조약돌 벽돌
-tile.cobblestone.3.desc=큰 조약돌 타일
-tile.cobblestone.4.desc=작은 조약돌 타일
+tile.cobblestone.3.desc=대형 조약돌 타일
+tile.cobblestone.4.desc=소형 조약돌 타일
 tile.cobblestone.5.desc=프랑스풍 조약돌 타일
 tile.cobblestone.6.desc=프랑스풍 조약돌 타일
 tile.cobblestone.7.desc=크리퍼 문양 조약돌 타일
-tile.cobblestone.8.desc=깨진 조약돌 타일
-tile.cobblestone.9.desc=큰 조약돌 타일
-tile.cobblestone.10.desc=크리퍼 조약돌 패널
-tile.cobblestone.11.desc=파인 조약돌
+tile.cobblestone.8.desc=손상된 조약돌 타일
+tile.cobblestone.9.desc=거대한 조약돌 타일
+tile.cobblestone.10.desc=크리퍼 문양 조약돌 패널
+tile.cobblestone.11.desc=오목한 조약돌
 tile.cobblestone.12.desc=조약돌 패널
 tile.cobblestone.13.desc=밝은 조약돌 패널
 tile.cobblestone.14.desc=어두운 조약돌 패널
 
-#Sandstone Blocks
+# 사암 (Sandstone Blocks)
 tile.chisel.sandstone.name=사암
 
-#Marble Blocks
+# 대리석 (Marble Blocks)
 tile.chisel.marble.name=대리석
-tile.marble.0.desc=대리석
+tile.marble.0.desc=가공 안된 대리석
 tile.marble.1.desc=대리석 벽돌
-tile.marble.2.desc=클래식 대리석 패널
+tile.marble.2.desc=고전적인 대리석 패널
 tile.marble.3.desc=화려한 대리석 패널
 tile.marble.4.desc=대리석 패널
 tile.marble.5.desc=대리석 블록
-tile.marble.6.desc=어두운 크리퍼 대리석
-tile.marble.7.desc=밝은 크리퍼 대리석
+tile.marble.6.desc=어두운 크리퍼 문양 대리석
+tile.marble.7.desc=밝은 크리퍼 문양 대리석
 tile.marble.8.desc=조각된 대리석
-tile.marble.9.desc=둥근 조각의 대리석
-tile.marble.10.desc=파인 대리석
-tile.marble.11.desc=크게 파인 대리석
-tile.marble.12.desc=대리석 타일
-tile.marble.13.desc=Arranged Marble Tiles
-tile.marble.14.desc=화려한 대리석 타일
-tile.marble.15.desc=대리석 블록
-
-#Marble Slabs
-tile.chisel.marbleSlab.name=대리석 반블럭
```

Comment:
You changed the localization a bunch of times like this.
marbleSlab->marble_slab

Was this intentional?  You did it several times